apps/studio/components/interfaces/Auth/Hooks/HooksListing.tsx

```diff
@@ -0,0 +1,191 @@
+  DropdownMenuTrigger,
+} from 'ui'
+
+import { FormHeader } from 'components/ui/Forms'
+import { useAuthConfigQuery } from 'data/auth/auth-config-query'
+import { useAuthConfigUpdateMutation } from 'data/auth/auth-config-update-mutation'
+import { useOrgSubscriptionQuery } from 'data/subscriptions/org-subscription-query'
+import { useCheckPermissions, useSelectedOrganization } from 'hooks'
+import { AlertCircle, ChevronDown } from 'lucide-react'
+import ConfirmationModal from 'ui-patterns/Dialogs/ConfirmationModal'
+import { CreateHookSheet } from './CreateHookSheet'
+import { HookCard } from './HookCard'
+import { HOOKS_DEFINITIONS, HOOK_DEFINITION_TITLE, Hook } from './hooks.constants'
+import { extractMethod, isValidHook } from './hooks.utils'
+
+export const HooksListing = () => {
+  const { ref: projectRef } = useParams()
+  const { data: authConfig, error: authConfigError, isError } = useAuthConfigQuery({ projectRef })
+  const organization = useSelectedOrganization()
+  const { data: subscription } = useOrgSubscriptionQuery(
+    { orgSlug: organization?.slug },
+    { enabled: IS_PLATFORM }
+  )
+
+  const [selectedHook, setSelectedHook] = useState<HOOK_DEFINITION_TITLE | null>(null)
+  const [selectedHookForDeletion, setSelectedHookForDeletion] = useState<Hook | null>(null)
+
+  const { mutateAsync: updateAuthConfig } = useAuthConfigUpdateMutation()
+  const canUpdateConfig = useCheckPermissions(PermissionAction.UPDATE, 'custom_config_gotrue')
+
+  if (isError) {
+    return (
+      <Alert_Shadcn_ variant="destructive">
+        <AlertCircle strokeWidth={2} />
+        <AlertTitle_Shadcn_>Failed to retrieve auth configuration</AlertTitle_Shadcn_>
+        <AlertDescription_Shadcn_>{authConfigError.message}</AlertDescription_Shadcn_>
+      </Alert_Shadcn_>
+    )
+  }
+
+  // configuration for all auth hooks.
+  const hooks: Hook[] = HOOKS_DEFINITIONS.map((definition) => {
+    return {
+      ...definition,
+      enabled: authConfig?.[definition.enabledKey] || false,
+      method: extractMethod(
+        authConfig?.[definition.uriKey] || '',
+        authConfig?.[definition.secretsKey] || ''
+      ),
+    }
+  })
+
+  const nonEnterpriseHookOptions = hooks.filter((h) => !isValidHook(h) && !h.enterprise)
+  const enterpriseHookOptions = hooks.filter((h) => !isValidHook(h) && h.enterprise)
+
+  return (
+    <>
+      <FormHeader
+        title="Auth Hooks"
+        description="Use Postgres functions or HTTP endpoints to customize the behavior of Supabase Auth to meet your needs."
+        actions={
+          <DropdownMenu modal={false}>
+            <DropdownMenuTrigger asChild>
+              <Button
+                type="default"
+                iconRight={<ChevronDown className="w-4 h-4" strokeWidth={1} />}
+              >
+                Add hook
+              </Button>
+            </DropdownMenuTrigger>
+            <DropdownMenuContent className="w-76 p-0" align="end">
+              <div className="p-2">
```

Comment:
`p-1`. DropdownContent uses `p-1` which I'm guessing it what your emulating

Reply:
FIxed.